Orange/data/variable.py

```diff
@@ -451,8 +452,14 @@ def __reduce__(self):
         # Use make to unpickle variables.
         return make_variable, (self.__class__, self._compute_value, self.name), self.__dict__
 
-    def copy(self, compute_value):
-        var = type(self)(self.name, compute_value=compute_value, sparse=self.sparse)
+    def rename(self, name):
```

Comment:
This name is misleading because it it implies that it renames a variable. I would eliminate this function and just use `copy` when I need a new variable with a different name.

If you really want to keep rename (which I really don't like, but it's a matter of opinion), we should call it `renamed`. (Like `list.sort`, which works in place, and `list.renamed`, which returns a new list.)